config/kubernetes/org.yaml

```diff
@@ -1659,9 +1670,7 @@ teams:
     repos:
       committee-security-response: admin
   stage-bots:
-    description: Bots that only have admin access to staged repositories. Giving admin
-      access to other repositories may allow the bot to automatically unintentionally
-      close issues in those repositories.
+    description: Bots that only have admin access to staged repositories. Giving admin access to other repositories may allow the bot to automatically unintentionally close issues in those repositories.
```

Comment:
Is this diff unintentional?

Reply:
Same question for the other similar ones above.

Reply:
it was. TBH - I'm not sure where it came from. I thought I only ran korg add commands unless it auto-formatted them Will poke around a bit

Reply:
I did a test and replicated the same behavior with just using the korg command
not sure how we didn’t see this before, so something has to have changed somewhere
```
git checkout main
git fetch upstream
git rebase upstream/main
git checkout -b 'format-test'
korg add  seifrajhi --org kubernetes --org kubernetes-sigs --confirm
```
(person was the first commit in my other branch)
checked diff and it mirrors what I pushed